scripts/cnode-helper-scripts/env

```diff
@@ -141,6 +142,8 @@ fi
 [[ -z "${CNODE_HOME}" ]] && CNODE_HOME=/opt/cardano/cnode
 
 CNODE_NAME="$(basename ${CNODE_HOME})"
+CNODE_VNAME=$(tr '[:upper:]' '[:lower:]' <<< ${CNODE_NAME//_HOME/})
+CNODE_VNAME_UC=$(tr '[:lower:]' '[:upper:]' <<< ${CNODE_VNAME})
```

Comment:
varname -> _UC may not be very obvious

Reply:
happy for suggestions 😄  .... would spelling it out like CNODE_VNAME_UPPERCASE be preferred?

Reply:
prolly, unless we have any other suggestion